.github/ISSUE_TEMPLATE/community-request-template.md

```diff
@@ -7,23 +7,28 @@ assignees: ''
 
 ---
 
-# Community Note
-* Please vote for this issue by adding a 👍 reaction to the issue. This will help the community and maintainers from the Amazon Chime SDK team to prioritize this request.
-* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request.
+## Community Note
+
+* Please vote for this issue by adding a 👍. Amazon Chime SDK team will use this info to prioritize the request. Comments like "+1" or "me too" are not recommended.
```

Comment:
Can we say somthing more like: 

`Please do not add additional comments such as "+1" or "me too" to existing issues - instead please vote on the issue by adding a 👍 to the issue. The Amazon Chime SDK team will prioritize the request based on the number of 👍 the issue has received.`

Reply:
Yeah, this is much better, will do